packages/applications-service-api/src/repositories/project.backoffice.repository.js

```diff
@@ -42,6 +42,9 @@ const getAllApplications = async (options = {}) => {
 		where['AND'].push({
 			OR: [
 				{ projectName: { contains: searchTerm } },
+				...(process.env.FEATURE_ALLOW_WELSH_TRANSLATION === 'true'
```

Comment:
We have a pattern to set the env variable on the config.js file which is then exported as a variable from there.